framework/filters/RateLimiter.php

```diff
@@ -61,26 +61,35 @@ class RateLimiter extends ActionFilter
      */
     public $response;
 
+    public function init()
+    {
+        if (!$this->request) {
```

Comment:
https://github.com/yiisoft/yii2/blob/master/framework/filters/RateLimiter.php#L75
It is difficult to say why in the filter these properties